blockatlas/blockatlas-new.md

```diff
@@ -0,0 +1,105 @@
+# Block Atlas Developer Guide
+
+## Overview
+
+Block Atlas provides access to blockchain data.
+It is used by Trust Wallet to query transaction information.
+
+A list of supported coins can be found [here](TODO://readme.md).
+
+## Integration
+
+While we are happy about any new coin integrations we cannot immediately accept every request. To keep our high security and reliability standards, every integration must be approved by the Trust Wallet team first.
+
+Pull requests other than coin integrations are always welcome. You can contact the Trust Wallet team over [TODO].
+
+To integrate an approved coin with Block Atlas, please follow this guide.
+
+### Requirements
+
+Before starting development on Block Atlas integration, make sure that _all_ of the following requirements are met:
+
```

Comment:
@terorie can you provide different transaction types? so it's easier to see how those work and what they do